mkosi/__init__.py

```diff
@@ -4280,7 +4280,10 @@ def install_unified_kernel(
                 option = "usrhash" if args.usr_only else "roothash"
                 boot_options = f"{boot_options} {option}={root_hash}"
             elif partlabel:
-                boot_options = f"{boot_options} root=PARTLABEL={partlabel}"
+                if args.usr_only:
+                    boot_options = f"{boot_options} mount.usr=/dev/disk/by-partlabel/{partlabel}"
+                else:
+                    boot_options = f"{boot_options} root=PARTLABEL={partlabel}"
```

Comment:
`mount.usr` takes the same options as `root`, hence please use `mount.usr=PARTLABEL` here as well, so the following:

```
option = "mount.usr" if args.usr_only else "root"
boot_options = f"{boot_options} {option}=PARTLABEL={partlabel}
```

Reply:
Fixed